Code/HighLevelAPI/src/main/java/ie/tcd/wayfinder/highlevel/navigation/NavigationController.java

```diff
@@ -25,20 +37,34 @@ public String index()
      * ResponseEntity<String>("No ROUTES", HttpStatus.BAD_REQUEST); }
      */
 
-    @GetMapping("/getRoute/start/{start}/destination/{destination}")
-    public ResponseEntity<String> FindRoute(@PathVariable String start, @PathVariable String destination)
+    @GetMapping("/navigation/start/{startLong}/{startLat}/destination/{destLong}/{destLat}/")
+    public ResponseEntity<String> FindRoute(@PathVariable Float startLong, @PathVariable Float startLat, @PathVariable Float destLong, @PathVariable Float destLat) throws IOException
```

Comment:
First letter of method name is in lower case in camelCase convention

Reply:
fixed